### PR TITLE
MM Blog article

### PR DESCRIPTION
**Pull request by javibravo**

### CODE REVIEW

**Review by sean-garvey — DISMISSED** [review]

File: article.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+We already had some ideas in mind and it was supposed to be a really simply service, after some discussions 
+and tests we decided to go for the 3rd option. Originally it was a service that allowed us to 
+send scheduled messages to SQS queues writing few lines in a YAML file, it was really useful for the team. 
+Initially we were trying to find a solution to schedule jobs for Preakness but at the same time we were 
+thinking farther. Meanwhile we were using the tool we realized how cool was the tool and the potential it had: 
+
+* What if it would be able to trigger different type of events?
+* Can it be detached from FINP architecture?
+ 
+It was how LambdaCron was worn.
```
Comment (sean-garvey) [inline article.md]: *born

**Comment by javibravo** [comment]
@sean-garvey, @nickagel, @vincetse, 

Open to hear suggestions to improve the article: things to include or remove.
Don't be shy to correct me if there are better ways to say something, I bet there many things can be improved.

**Comment by javibravo** [comment]
@fsargent ,
I am about to open source LambdaCron. I have been working in an article for MM blog. I would like you to review it and suggest any change/update if required.

**Comment by javibravo** [comment]
@fsargent , I have added some small changes: https://github.com/MediaMath/lambda-cron/pull/28/commits/ac250ad23fc952d6b83530ea59abf7b7629d7029

@fsargent , @vincetse , 
I would like a final review (and approval if proceed) from you. After that I will merge and publish.
Thanks.

File: article.md [hunk 2]
```diff
@@ -109,6 +109,6 @@ The job is scheduled to run at 5 pm on the 5th every month. The **type** of the
 
 Before creating tasks, a LambdaCron environment must be deployed by following the instructions in the [README](https://github.com/mediaMath/lambda-cron) of the project.
 
-LambdaCron CLI also provides commands that can automate tasks. E.G. validate and upload tasks to a LambdaCron environment using  Continuous Integration tools. We will follow up with more examples and how to work with LambdaCron in future blog posts.
+LambdaCron CLI also provides commands that can automate development and maintenace tasks. E.G. validate and upload tasks to a LambdaCron environment using  Continuous Integration tools. We will follow up with more examples and how to work with LambdaCron in future blog posts.
```
Comment (fsargent) [inline article.md]: maintenance*

File: article.md [hunk 3]
```diff
@@ -14,7 +14,7 @@ The most simple and straightforward way to schedule task is using Linux [crontab
 
 ### 2. AWS CloudWatch Events
 
-AWS configures events using [CloudWatch](http://docs.aws.amazon.com/AmazonCloudWatch/latest/events/WhatIsCloudWatchEvents.html), to invoke other AWS services in a cron-like way. As the team's infrastructure is based on AWS, using AWS services to schedule events made total sense. Preakness also uses Simple Queue Service (SQS) to queue jobs integrated with CloudWatch events. It seemed like a good option, however we found some negative aspects we didn't feel comfortable with:
+AWS configures events using [CloudWatch](http://docs.aws.amazon.com/AmazonCloudWatch/latest/events/WhatIsCloudWatchEvents.html), to invoke other AWS services in a cron-like way. As the team's infrastructure is based on AWS, using AWS services to schedule events made total sense. Preaknes uses Simple Queue Service (SQS) to queue jobs, service which is integrated with CloudWatch events. It seemed like a good option, however we found some negative aspects we didn't feel comfortable with:
```
Comment (fsargent) [inline article.md]: This is a negative change. 
You don't need ",service which is" 
Preakness should have two 's' to be consistent with the rest of the article.

Revert this change.

**Review by fsargent — CHANGES_REQUESTED** [review]

**Comment by javibravo** [comment]
@fsargent , fixed.

**Review by fsargent — APPROVED** [review]

**Review by sean-garvey — APPROVED** [review]

**Pull request merged**